t/t0024-content-s3.t

```diff
@@ -139,6 +163,73 @@ test_expect_success HAVE_JQ 'kvs-checkpoint.get foo returns same value' '
         test_cmp value2.exp value2.out
 '
 
+test_expect_success 'config: reload config does not take effect immediately' '
+	flux config reload 2>/dev/null &&
+	flux dmesg | grep content-s3 | tail -1 >reload.log &&
+	grep "changes will not take effect until next flux restart" reload.log
+'
+
+test_expect_success 'save good config' '
+	mv content-s3.toml content-s3.save &&
+	mv creds/creds.toml creds/creds.save
```

Comment:
If you move the creds file out of the way then the expected failures in subsequent tests could be masked by the missing cred file.  Suggest you copy not move it.